vignettes/parallel.Rmd

```diff
@@ -0,0 +1,218 @@
+
+That said, Hipercow offers a few options for limiting how long your tasks can run for, specifying when your tasks can run on the cluster, and also allows you to politely allow other tasks to take priority over yours. 
+
+## The maximum runtime
+
+If you know how long your task should take, and you'd like to abort if it takes longer, then use the `max_runtime` argument when requesting your resources. You can specify an integer number of minutes, or strings involving the letters `d`, `h` and `m`, for days, hours and minutes, such as `"40d`" or `"1h30"`.
+
+This might be useful if you have stochastic fitting tasks that might not be converging, and you'd like a time limit after which the tasks are aborted. Or perhaps you have a task that you'd only like to run for a while to check that the early stages look good.
+
+## Delaying tasks starting
+
+If you are about to launch a large number of very time consuming tasks, that are not crucially urgent, it may be helpful to others if they could start running on the cluster outside of working hours. You can do this by setting the `hold_until` argument for `hipercow_resources`. A number of formats are allowed:-
+
+* An integer represents a number of minutes.
+* Strings in the form `"5h`" or `"1h30`" or `"2d`" can delay for hours, minutes or days.
+* R's Date type can be used to indicate midnight on the given date.
+* R's POSIXt type a date and time to be represented.
+* `"tonight`" makes a task wait until after 7pm this evening before starting.
+* `"midnight`" delays a task until tomorrow begins.
+* `"weekend`" delays a task until midnight on Saturday.
+
+## Lowering your priority
+
+If you are about to launch a large number of tasks, another way of being polite to your colleagues is to set `priority='low'` in `hipercow_resources`. This allows tasks lower down the queue with `normal` priority to overtake your `low` priority tasks and run on available resources first. Effectively, it means you can launch large volumes of tasks without annoying people, getting all the available resources, but without holding others up very much if they also need something to run.
+
+This never causes your running tasks to get cancelled; it is only relevant when there are available resources on a node for the scheduler to consider which tasks to allocate those resources too. Therefore, low priority works best if additionally your tasks don't take too long to run, so there are reasonably frequent opportunities for the scheduler to decide what to do.
+
+```{r parallel_task}
+resources <- hipercow_resources(max_runtime = "1h")
```

Comment:
this is in the wrong place?

Reply:
Oops, yes, that's leftover - but will add better examples earlier where it should go